Dockerfile

```diff
@@ -1,23 +1,30 @@
-FROM python:2.7-onbuild
-
-# 1) COPY requirements.txt
-# 2) Run pip install on ^
-# 3) COPY . /usr/src/app
-
-#ENV GUNICORN_VERSION 19.3.0
+FROM python:2.7
 
 ## Install our dependencies
-
-RUN apt-get update && \
-    apt-get -y install libxml2-dev libxslt1-dev zlib1g-dev openssh-client build-essential && \
-#    pip install gunicorn==${GUNICORN_VERSION} && \
-    apt-get -y autoremove && \
-    apt-get clean
+RUN apt-get update \
+    && apt-get -y install \
+        libxml2-dev \
+        libxslt1-dev \
+        zlib1g-dev \
+        openssh-client \
+        build-essential \
+        texlive-latex-base \
+        texlive-latex-extra \
+        texlive-latex-recommended \
+        texlive-fonts-recommended \
+    && apt-get -y autoremove \
+    && apt-get clean
+
+WORKDIR /usr/src/app/readthedocs.org
 
 ## Apply our own overrides
-#COPY local_settings.py /usr/src/app/readthedocs.org/readthedocs/settings/local_settings.py
+COPY local_settings.py.example /usr/src/app/readthedocs.org/readthedocs/settings/local_settings.py
+COPY entrypoint.sh ./
```

Comment:
Because I stopped using the `-onbuild` version which does work at build time that's implicit, and essentially hidden to the user. 

One step it did was to `# 3) COPY . /usr/src/app` so now we have to do this manually. I also am doing this in specific steps to keep a good working docker cache so rebuilds are quick.